Web/beeminder_one_goal.1m.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" python3
+# -*- coding: utf-8 -*-
+
+# Metadata allows the plugin to show up in the xbar app and website.
+#
+#  <xbar.title>Beeminder</xbar.title>
+#  <xbar.version>v0.1</xbar.version>
+#  <xbar.author>Tom Adamczewski</xbar.author>
+#  <xbar.author.github>tadamcz</xbar.author.github>
+#  <xbar.desc>Concisely show the status of a Beeminder goal</xbar.desc>
+#  <xbar.image>https://images2.imgbox.com/8d/2d/wLOcD9sz_o.png</xbar.image>
+#  <xbar.dependencies>python</xbar.dependencies>
+
+# <xbar.var>string(VAR_USERNAME='johnsmith'): Your Beeminder username</xbar.var>
+# <xbar.var>string(VAR_AUTH_TOKEN=''): Your Beeminder auth token</xbar.var>
+# <xbar.var>string(VAR_GOAL_SLUG='exercise'): Your Beeminder goal slug e.g. 'exercise' for beeminder.com/myusername/exercise</xbar.var>
+# <xbar.var>string(VAR_GOAL_EMOJI='🏋️'): Emoji to represent your goal (will appear in the menu bar)</xbar.var>
+
+import os
+import json
+import time
+from urllib.request import urlopen, Request
+from urllib.parse import urlencode
+from urllib.error import URLError
+
+# Script-level variables
+USERNAME = os.environ["VAR_USERNAME"]
+AUTH_TOKEN = os.environ["VAR_AUTH_TOKEN"]
```

Comment:
When the plugin is first run, this line causes a failure where it has no default.

<img width="948" alt="Screenshot 2023-04-06 at 10 12 47" src="https://user-images.githubusercontent.com/5545555/230404015-5d91a889-f84b-4f7d-a023-7b4b5b243ff2.png">

Would be good to have some sort of default or have this fall through to an empty string.